dask/bytes/core.py

```diff
@@ -483,6 +483,13 @@ def ensure_protocol(protocol):
                         "    or\n"
                         "    pip install s3fs")
 
+    elif protocol in ('gs', 'gcs'):
+        import_required('gcsfs',
```

Comment:
The import will inject into the _filesystems dict - is that correct? The check is only really needed is gcs is not already in the dict, but I guess it has no overhead.

Reply:
I'm mostly just copying the behavior for `s3fs`.  This is the only place where I see that library listed explicitly.

Reply:
OK, I'm happy with it.